src/sfCommand.ts

```diff
@@ -345,6 +345,9 @@ export abstract class SfCommand<T> extends Command {
   }
 
   public async _run<R>(): Promise<R> {
+    process.on('SIGINT', () => {
```

Comment:
ref: https://nodejs.org/api/process.html, scroll down to `Signal events`

while we're here, does it make sense to handle the other handleable things (`SIGTERM`, `SIGBREAK` and possibly `SIGHUP` for windows?)